Makefile

```diff
@@ -7,13 +7,6 @@ TARGETS := $(shell ls scripts)
 	@./.dapper.tmp -v
 	@mv .dapper.tmp .dapper
 
-$(TARGETS): .dapper
```

Comment:
Line 10/11 seem to be in the file before we made changes? https://github.com/rancher/kubernetes-package/pull/138/commits/8998f276c8d8bae1b867439bbb67951909fdc50e#diff-b67911656ef5d18c4ae36cb6741b7965L10